cognite/neat/graph/queries/_base.py

```diff
@@ -74,30 +74,33 @@ def list_instances_of_type(self, class_uri: URIRef) -> list[ResultRow]:
         # Select queries gives an iterable of result rows
         return cast(list[ResultRow], list(self.graph.query(query)))
 
-    def triples_of_type_instances(self, rdf_type: str) -> list[tuple[str, str, str]]:
+    def triples_of_type_instances(self, rdf_type: str | URIRef) -> list[tuple[str, str, str]]:
```

Comment:
Small extension of this method, used in experimentation. We can remove it, but I think it is an ok extension to include.